streaming/src/main/scala/org/apache/spark/streaming/scheduler/JobScheduler.scala

```diff
@@ -66,6 +66,12 @@ class JobScheduler(val ssc: StreamingContext) extends Logging {
     }
     eventLoop.start()
 
+    // Estimators receive updates from batch completion
```

Comment:
This comment is not relevant. Shouldnt it be "Attach the rate controllers of the input dstreams for receiving batch info updates"?